test/cppsuite/tests/hs_cleanup.cpp

```diff
@@ -47,6 +47,31 @@ class hs_cleanup : public test {
         init_operation_tracker();
     }
 
+    /*
+     * The checkpoint function has been overridden to accommodate the offloaded checkpoint cleanup
+     * thread, enabled by the debug=checkpoint_cleanup options, to accurately measure checkpoint
+     * cleanup pages removed.
+     */
```

Comment:
We don't do this to accurately measure the stat, we do this because the checkpoint cleanup thread runs every 5 minutes and the default version of the test only runs for 60 seconds.
What do you think of simply updating the [connection cfg string](https://github.com/wiredtiger/wiredtiger/blob/395866cec4676ce735450180a5dc992ed8e0b1dd/dist/api_data.py#L562-L564) instead of overriding checkpoint?

Reply:
Yeah, passing the "checkpoint_cleanup" configuration from the evergreen.yml file seems to be a cleaner approach. Fixed in c36cf5f2c7a55d7270ccace87e0024b8c8726583